### PR TITLE
req 3 metafiles

### PR DESCRIPTION
**Pull request by anerishah126**

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+---
+# This manifest deploys a Python Flask application with a Cloudant database
+applications:
+- name: nyu-shopcart-service-sp2022
```
Comment (ceciliastern) [inline manifest.yml:5]: 2021?

File: manifest.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  path: .
+  instances: 2
+  memory: 64M
+  routes:
+  - route: nyu-shopcart-service-sp2022.us-south.cf.appdomain.cloud
```
Comment (ceciliastern) [inline manifest.yml:10]: here too

File: manifest.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  disk_quota: 1024M
+  buildpacks: 
+  - python_buildpack
+  timeout: 180
+  services:
+  - Cloudant-as
```
Comment (ceciliastern) [inline manifest.yml:16]: think this should be ElephantSQL

**Review by anerishah126 — COMMENTED** [review]
addressed comments

**Pull request merged**